docs/content/dagster-cloud/account/authentication/utilizing-scim-provisioning.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Utilizing SCIM provisioning in Dagster Cloud | Dagster Docs"
+
+display_name: "SCIM"
+feature_name: "scim"
+pricing_plan: "enterprise"
+---
+
+# Utilizing SCIM provisioning in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+The [System for Cross-domain Identity Management specification](https://scim.cloud/) (SCIM) is a standard designed to manage user identity information. With SCIM, you can:
+
+- Centralize user management in your Identity Provider (IdP)
+- Sync users and groups to Dagster Cloud from your IdP
+- TODO?
+
+---
+
+## Understanding SCIM provisioning
+
+SCIM provisioning eases the burden of manually provisioning users across your cloud applications. When enabled, you can automatically sync user information from your IdP to Dagster Cloud and back again, ensuring user data is always up-to-date.
+
+For a detailed look at SCIM provisioning, [check out this blog post](https://www.strongdm.com/blog/scim-provisioning).
+
+### Managing users
+
+When SCIM is enabled in Dagster Cloud, a few things about user management will change:
+
+- **New users must be added in the IdP.** The ability to add new users will be disabled in Dagster Cloud while SCIM is enabled.
+- **Updates to users' emails and names must be completed in the IdP.** When these attributes are changed in the IdP, they'll automatically be synced to Dagster Cloud.
+- **Only 'unsynced' users can be removed in Dagster Cloud.** 'Synced' users will have an icon indicating they're externally managed, while unsynced users will not. For example, the first two users in the following image are synced, while the last isn't:
+
+  <Image
+  alt="Highlighted unsynced user in the Dagster Cloud UI"
+  src="/images/dagster-cloud/sso/scim-unsynced-user.png"
+  width={732}
+  height={376}
+  />
+
+### Managing teams
```

Comment:
I'm struggling a bit with this section after going through our Slack thread about it. Is there anything else to add or clarify here?